client/packages/sprotty-client/src/base/model/update-model-command.ts

```diff
@@ -38,13 +38,12 @@ import { GLSP_TYPES } from "../../types";
 *  UpdateModel behaves the same as SetModel if no model is present yet.*/
 @injectable()
 export class SetModelActionHandler implements IActionHandler {
+    handledActionKinds = [SetModelCommand.KIND];
```

Comment:
`handledActionKinds=....` can be removed. Looks like a leftover from the old `SelfInitializingActionHandler`